README.md

```diff
@@ -141,11 +143,13 @@ When ingesting a metric, the following actions occur:
 
 ### Data Query
 
-The goals of a data query are
```

Comment:
having 2 goals here confused me.

it seems more clear to instead say there is one goal with two required steps.  what do you think?

Reply:
Thank you! I struggled with that particular line in the README and your expanded verbage is much better.